modules/videopress/js/editor-view.js

```diff
@@ -209,8 +234,21 @@
 					}, renderer );
 
 					editor.insertContent( wp.shortcode.string( args ) );
+				},
+				onopen : function ( e ) {
```

Comment:
It may be better to move my overrides from modifying the FormItem.renderHtml prototype above into this.